controller/src/main/java/org/jboss/as/controller/extension/ExtensionRegistry.java

```diff
@@ -209,7 +209,7 @@ public ExtensionRegistry build() {
     }
 
     private final ProcessType processType;
-    private final Stability stability;
+    private volatile Supplier<Stability> stability;
```

Comment:
This should be final, not volatile, no?

Reply:
@pferraro You're right, this volatile was from a previous attempt